src/main/scala/chiseltest/backends/treadle/TreadleBackend.scala

```diff
@@ -14,7 +16,8 @@ class TreadleBackend[T <: Module](
   val dut:                T,
   val dataNames:          Map[Data, String],
   val combinationalPaths: Map[Data, Set[Data]],
-  tester:                 TreadleTester)
+  tester:                 TreadleTester,
+  coverageAnnotations:    AnnotationSeq)
```

Comment:
Does it make sense for something more general here? (eg, pass in all annotations)

Reply:
My assumption was that this is a semi-private API, i.e., most users aren't really expected to manually instantiate it. Thus changing this to be more broad in the future shouldn't be too big of a problem. For now this only contains the coverage annotations, so the name is very descriptive wrt. the current implementation.

Reply:
Yep, that's fine.